earthpy/plot.py

```diff
@@ -20,11 +20,12 @@ def colorbar(mapobj, size="3%", pad=0.09, aspect=20):
 
     Parameters
     ----------
-    mapobj : the matplotlib axes element.
-    size : char
-        The percent width of the colorbar relative to the plot. default = 3%
-    pad : int
-        The space between the plot and the color bar. Default = .09
+    mapobj : matplotlib axis object
+        The image that the colorbar will be representing as a matplotlib axis object.
+    size : char (optional)
+        The percent width of the colorbar relative to the plot.
+    pad : int (optional)
```

Comment:
hey @mirob9363 It's good pracice to include the default values of parameters when they exist. in this case the defaul size = 3% and the default pad = .09. Will you please add those values back? Imagine someone using this function - it's very good to see what the default values are!

Reply:
Also - it does appear as if aspect is never used in this function. We could chose to remove it from the function OR we could add it to the return statement when colorbar is called. @joemcglinchy do you have any thoughts on using the aspect argument for colorbars? do you ever use it? i haven't used it but could see it being useful.